.circleci/config.yml

```
@@ -0,0 +1,20 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run: pip install .
+      - run: pip install mock pylint
+      - run:
+          command: python -m unittest discover
+          name: Run Unit Test
```

Minor nitpick, but do we need the `Run` prefix here, them being run steps and all?
```suggestion
          name: Unit Tests
```